cluster/aws/config-default.sh

```diff
@@ -82,6 +82,8 @@ ENABLE_CLUSTER_DNS="${KUBE_ENABLE_CLUSTER_DNS:-true}"
 DNS_SERVER_IP="10.0.0.10"
 DNS_DOMAIN="cluster.local"
 DNS_REPLICAS=1
+OPENCONTRAIL_PUBLIC_SUBNET="${OPENCONTRAIL_PUBLIC_SUBNET:-10.1.0.0/16}"
+OVERLAY_NETWORK_PROVIDER="${OVERLAY_NETWORK_PROVIDER:-none}" # opencontrail
```

Comment:
This should probably be "NETWORK_PROVIDER". Whether it is overlay or not... there will be non-overlay options (e.g. VLANs, calico).